README.md

```diff
@@ -64,17 +64,9 @@ Flatiron School Portal account to your GitHub account.
 1. Open the [Flatiron School Student Portal webpage][flatiron portal]
    (https://portal.flatironschool.com) _(Note: you need to be logged in to
    Flatiron School Student Portal.)_
-2. Click on "Course" in the navigation bar at the top of the screen
-3. Click the blue "Switch Materials" button in the dropdown
-4. Click on the course that you are about to start
-5. Open the [GitHub Account Management webpage][github account]
-   (https://learn.co/account/github) _(Note: you may be asked to log in. Use
-   your Flatiron School Student Portal username and password here.)_
-   <!-- Note: this domain is not the Portal because of Canvas flows -->
-6. Connect your GitHub account to your Flatiron School Portal account
+2. Connect your GitHub account to your Flatiron School Portal account
```

Comment:
Once some one has logged into Portal, it might be difficult for them to find _where_ to connect there Github account. Maybe we could add a couple of extra steps:
```suggestion
2. Click on your user icon in the top right corner
3. Select Account Management from the tabs along the top of the page
4. Connect your GitHub account to your Flatiron School Portal account
```
Or you could re-used the text from below instructing how to get to the page with the OAuth tokens perhaps?

Reply:
> Once some one has logged into Portal, it might be difficult for them to find _where_ to connect there Github account. Maybe we could add a couple of extra steps:

This is really great, I didn't think of that. But one thing before I commit this. In the **third step** says: Select **Account Management** from the tabs along the top of the page.
This is a bit confusing since there is no **Account Management** directly to click on.

What do you think of these changes?
1. Open the [Flatiron School Student Portal webpage][flatiron portal]
   (https://portal.flatironschool.com) _(Note: you need to be logged in to
   Flatiron School Student Portal.)_
2. Click on your user icon in the top right corner
_(Note: To reduce the work from step 3 to 6 you can click [Account Management][Flatiron Account Management] then continue to the rest of the steps.)_
3. On the top right-hand corner click on your profile picture (This will open a drop-down menu).
4. Click on Base.
5. Again on the top right-hand corner click on the profile picture.
6. Click on Account Management
7. Click on Create Account if you don't have one, or if you have a Github account then click on Connect Account.

By the way, I'm not really good at Open Sourcing since I started recently, sorry if it is not organized.

Reply:
You are totally right. Those instructions look great.

The instructions I was writing were if you are already in Base. The click on "Account Management" that I was thinking of is what you have on step 6.

And you are not disorganized at all. This is fantastic :)

Also, if you don't mind me asking, where are you seeing these instructions or how were you directed to them?

Reply:
> The instructions I was writing were if you are already in Base. The click on "Account Management" that I was thinking of is what you have on step 6.

Oh, I didn't realize that.

> Also, if you don't mind me asking, where are you seeing these instructions or how were you directed to them?

Sorry, I forgot to link to the section since there was no template :D .The instructions are on the **Canvas**. I don't know if you can access it but it is here:
[Windows](https://my.learn.co/courses/146/pages/configuring-git-github-and-the-flatiron-student-portal-on-wsl2?module_item_id=19470) and [macOS](https://my.learn.co/courses/146/pages/configuring-git-github-and-the-flatiron-student-portal-on-macos?module_item_id=19514)
Since I use Windows I only checked Windows instructions but I just realized that macOS has the same instructions as Windows.

Reply:
Thank you so much for this fix @RawandDev ! We’ve merged your changes in.